<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerArmaturesModule</class>
 <widget class="qSlicerWidget" name="qSlicerArmaturesModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Armatures</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="ctkCheckBox" name="ArmatureVisibilityCheckBox">
     <property name="text">
      <string>Armature:</string>
     </property>
     <property name="indicatorIcon">
      <iconset resource="../../../../../../../../Slicer/Slicer4/Slicer4/Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc">
       <normaloff>:/Icons/VisibleOn.png</normaloff>
       <normalon>:/Icons/VisibleOff.png</normalon>:/Icons/VisibleOn.png</iconset>
     </property>
     <property name="indicatorIconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="ArmatureDisplayCollapsibleButton">
     <property name="text">
      <string>Armature Display</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="BonesCollapsibleButton">
     <property name="text">
      <string>Bones</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="1" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="HierarchyCollapsibleGroupBox">
        <property name="title">
         <string>Hierarchy</string>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="ParentBoneLabel">
           <property name="text">
            <string>Bone Parent:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="qMRMLNodeComboBox" name="ParentBoneNodeComboBox">
           <property name="nodeTypes">
            <stringlist>
             <string>vtkMRMLBoneNode</string>
            </stringlist>
           </property>
           <property name="noneEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="LinkedToParentLabel">
           <property name="text">
            <string>Linked to Parent:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="LinkedToParentCheckBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_2">
        <property name="title">
         <string>Positions</string>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="HeadLabel">
           <property name="text">
            <string>Head:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ctkCoordinatesWidget" name="HeadCoordinatesWidget"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="TailLabel">
           <property name="text">
            <string>Tail:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ctkCoordinatesWidget" name="TailCoordinatesWidget"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox">
        <property name="title">
         <string>Bone Display</string>
        </property>
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="BoneNameLabel">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="BoneNameLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="BoneVisibilityLabel">
           <property name="text">
            <string>Visibility:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="BoneVisibilityCheckBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="BoneRepresentationLabel">
           <property name="text">
            <string>Representation:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="BoneRepresentationComboBox">
           <item>
            <property name="text">
             <string>Line</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cylinder</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Octohedron</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="qMRMLAnnotationTreeView" name="BonesTreeView">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLArmatureNode</string>
          <string>vtkMRMLBoneNode</string>
         </stringlist>
        </property>
        <property name="showScene">
         <bool>false</bool>
        </property>
        <property name="showRootNode">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLNodeComboBox" name="ArmatureNodeComboBox">
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLArmatureNode</string>
      </stringlist>
     </property>
     <property name="renameEnabled">
      <bool>true</bool>
     </property>
     <property name="baseName">
      <string>Armature</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLTreeView</class>
   <extends>QTreeView</extends>
   <header>qMRMLTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLAnnotationTreeView</class>
   <extends>qMRMLTreeView</extends>
   <header>qMRMLAnnotationTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCoordinatesWidget</class>
   <extends>QWidget</extends>
   <header>ctkCoordinatesWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../../../../Slicer/Slicer4/Slicer4/Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerArmaturesModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ParentBoneNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ArmatureNodeComboBox</sender>
   <signal>currentNodeChanged(bool)</signal>
   <receiver>ArmatureDisplayCollapsibleButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ArmatureNodeComboBox</sender>
   <signal>currentNodeChanged(bool)</signal>
   <receiver>BonesCollapsibleButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerArmaturesModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ArmatureNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerArmaturesModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>BonesTreeView</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ArmatureNodeComboBox</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>BonesTreeView</receiver>
   <slot>setRootNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
